ansible/group_vars/all.yml

```diff
@@ -36,8 +36,8 @@ s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
 s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_port }}"
 
 # To use these vars without deploying the riak control machine
-# add s3_access_key and s3_secret_key to the ansible secret config directory
-s3_blob_db_access_key: "{{ s3_access_key if s3_access_key is defined else (hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '') }}"
-s3_blob_db_secret_key:  "{{ s3_secret_key if s3_secret_key is defined else (hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '') }}"
+# add riakcs_s3_access_key and riakcs_s3_secret_key to the ansible secret config directory
+s3_blob_db_access_key: "{{ riakcs_s3_access_key if riakcs_s3_access_key is defined else (hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '') }}"
+s3_blob_db_secret_key:  "{{ riakcs_s3_secret_key if riakcs_s3_secret_key is defined else (hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '') }}"
```

Comment:
becuase we also use amazon s3 so it's confusing to have 2 sets of s3 keys differing only in case: `s3_access_key` and `S3_ACCESS_KEY`